lib/Builder.js

```diff
@@ -96,7 +96,36 @@ class Builder {
         return this.metadata;
     }
     /**
-     * Builds a specific metadata file by name
+     * Build a template based on a list of metadata files in the retrieve folder.
+     *
+     * @param {string} businessUnit references credentials from properties.json
+     * @param {string} selectedType supported metadata type
+     * @param {string[]} keyArr customerkey of the metadata
+     * @param {string} market market localizations
+     * @returns {Promise.<TYPE.MultiMetadataTypeList>} -
+     */
+    static async buildTemplate(businessUnit, selectedType, keyArr, market) {
```

Comment:
_Messing with your head since 2018 ;-)_

I'll rename them a little... static vs non-static is a viable option but maybe we need to consider turning the builder and Retriever classes into all static. Don't really see the benefit here

Reply:
added "_" as prefix to non-static ones... 
for buildTemplate and buildDefinitition.
As I said, I believe we should convert this class into static only going forward.